metabrainz/templates/admin/users/edit.html

```diff
@@ -0,0 +1,240 @@
+{% extends 'admin/master.html' %}
```

Comment:
Was there no old template that did most of this before? Looks like a new file, but we had this feature before and I don't see any files being deleted.